### DISCUSSION TITLE
calendar doesnt work in foreign country

### DISCUSSION THREAD

**Discussion by chlgbdud** [nhn/tui.calendar#1271] [Q&A] [answered]
calendar that created in south korea is work 

but i stay in berlin for business trip, and i saw calendar but time data is wrong. 

what can i do?

**Comment by adhrinae** [comment, marked as answer]
@chlgbdud 

You need to set the proper timezone to serve the calendar globally.

https://github.com/nhn/tui.calendar/blob/main/docs/ko/apis/options.md#timezone
